packages/core/src/components/tag/stories/tag.core.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import { Meta, StoryObj } from '@storybook/web-components'
+
+import { html } from 'lit'
+
+import { TagStoryArgs } from './tag.args'
+
+export default {
+  title: 'Components/Tag',
+  ...TagStoryArgs,
+} as Meta
+
+const createTag = (args) => {
+  return html`
+    <atom-tag type="${args.type}" icon="${args.icon}"> ${args.label} </atom-tag>
+  `
+}
+
+export const Success: StoryObj = {
+  render: (args) => createTag(args),
+  args: {
+    type: 'success',
+    label: 'Promoção',
```

Comment:
(block) as this is a public project, is important follows writing all content using english

Reply:
@felipefialho done here https://github.com/juntossomosmais/atomium/pull/261/commits/f387db24e1fdecf8d3b28df949f21e0d42d25d5d